### ISSUE TITLE
Width / Height parameters don't change window size

### ISSUE THREAD

**Issue by DoctorRu** [dei79/jquery-modal-rails#16]
Width / height parameter don't affects the modal size.

<%= link_to_modal 'New User Modal Sized', new_user_path, :width=> "600", :height =>"400", :remote => true %><BR/>

or

<%= link_to_modal 'New User Modal Sized', new_user_path, :width=> "950", :height =>"800", :remote => true %><BR/>

Keeps the same size.

**Comment by lulalala** [comment]
Yep same here. However I am wondering if this is part of the interface (though it is in the demo page).

**Comment by lunarose23** [comment]
Same here...
